### PR TITLE
Relax `base` upper and bring CI to latest and greatest

### PR DESCRIPTION
**Pull request by mjgpy3**
Sorry it's kind of a grab-bag, but nothing's too wild here.

### CODE REVIEW

**Comment by mjgpy3** [comment]
Circle's not too happy about this change :laughing: :rofl:

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,25 +0,0 @@
-version: 2.1
```
Comment (z0isch) [inline .circleci/config.yml:1]: Does this mean bye bye for circle for this project!?

**Review by z0isch — APPROVED** [review]
I'm a bit hazy on the details for the github action stuff, but AFAICT it looks good!

File: .circleci/config.yml (hunk 1 above)
Comment (mjgpy3) [reply .circleci/config.yml:1]: Yep, indeed!

**Pull request merged**